### ISSUE TITLE
[Accessibility]Color contrast ratio of Collapsed icon of Font node is less than 3:1

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#4356]
* .NET Core Version: 
Net 6.0.100-alpha.1.20610.2

* Have you experienced this same bug with .NET Framework?: 
Yes

**More Info:**
1.	This issue can also repro on the below scenario:
Click Tools->Options-> Text Editor-> C/C++ -> Code Style-> Linter, then click ‘Checkers’ to collapse it, and observe the Color contrast of collapsed icon.
2.	This issue cannot repro on the expand icon of Font node.
3.	There are 2 similar bugs: [1257160](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1257160), [1236121](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1236121).

**Minimal repro:**
1.	Install the Accessibility Insights from https://accessibilityinsights.io/en/downloads.
2.	Extract ‘CollapsedIcon_PropertyGrid_Core.zip’ and open it in VS.
[CollapsedIcon_PropertyGrid_Core.zip](https://github.com/dotnet/winforms/files/5677354/CollapsedIcon_PropertyGrid_Core.zip)
3.	Build and run this project.
4.	Open the Accessibility Insights and select the Contrast of the directory on the left.
5.	Find the Font node, click on the color picker under Color2 and focus on the background of the collapsed icon.
6.	Click on the color picker under Color1 and focus on the collapsed icon.

**Expected behavior:**
Color contrast ratio of Collapsed icon of Font node should be greater than 3:1.

**Actual behavior:**
Color contrast ratio of Collapsed icon of Font node is 2.411:1, it is less than 3:1.
![Core_actual](https://user-images.githubusercontent.com/45864017/101870380-72bfe080-3bbc-11eb-8d7f-c092d82bdd67.png)

**Comment by RussKie** [comment]
@merriemcgaw we need guidance on colors taking in considerations WCAG2.1 guidelines.

**Comment by merriemcgaw** [comment]
This will go on our backlog for the accessibility work. I think we need to just keep this character the same color as the active text. @SergeySmirnov-Akvelon when you guys have the time this should be on our radar.

**Comment by Amy-Li03** [comment]
Verified this issue with .NET SDK 7.0.100-alpha.1.21479.26, this issue is fixed.
![image](https://user-images.githubusercontent.com/62929087/135407424-e60925ab-382a-4c55-8ffe-1256ad638da3.png)
